examples/pyproject.toml

```diff
@@ -365,13 +365,13 @@ confidence = ["HIGH", "CONTROL_FLOW", "INFERENCE", "INFERENCE_FAILURE", "UNDEFIN
 # --enable=similarities". If you want to run only the classes checker, but have
 # no Warning level messages displayed, use "--disable=all --enable=classes
 # --disable=W".
-disable = ["raw-checker-failed", "bad-inline-option", "locally-disabled", "file-ignored", "suppressed-message", "useless-suppression", "deprecated-pragma", "use-implicit-booleaness-not-comparison-to-string", "use-implicit-booleaness-not-comparison-to-zero", "use-symbolic-message-instead"]
+disable = ["raw-checker-failed", "bad-inline-option", "locally-disabled", "file-ignored", "suppressed-message", "useless-suppression", "deprecated-pragma", "use-symbolic-message-instead"]
 
 # Enable the message, report, category or checker with the given id(s). You can
 # either give multiple identifier separated by comma (,) or put this option
 # multiple time (only on the command line, not in the configuration file where it
 # should appear only once). See also the "--disable" option for examples.
-# enable =
+enable = ["c-extension-no-member"]
```

Comment:
Was this discussed on a PR? I can't find it. I think I'm not in favor of adding this to the example.

Reply:
It's automated from tbump, not sure why it's modified now.

Reply:
Locally running the commands I'm clean:

```
jwalls@Jacobs-MacBook-Air-4 pylint % python3.11 -m pylint --rcfile=/dev/null --generate-toml-config > out.toml
jwalls@Jacobs-MacBook-Air-4 pylint % cat out.toml| grep 'enable'                                              
# even if score is above --fail-under value. Syntax same as enable. Messages
# specified are enabled, while categories only check already-enabled messages.
# When enabled, pylint would attempt to guess common misconfiguration and emit
# everything first and then re-enable specific checks. For example, if you want
# --enable=similarities". If you want to run only the classes checker, but have
# no Warning level messages displayed, use "--disable=all --enable=classes
# enable =
```

Reply:
Do you think it's worth manually reverting these changes?

Reply:
I've redone it from zero considering the changes from the main branches and did not encounter the issue again. Maybe due to my local install that is sometime strangely setup to test for particular scenarios.